### DISCUSSION TITLE
Can't initialize EditorJS

### DISCUSSION THREAD

**Discussion by msrumon** [codex-team/editor.js#2024] [Q&A] [answered]
For some reasons, I can't make EditorJS work. [Here](https://codepen.io/msrumon/pen/XWVOORB)'s my attempt. Let me know what I'm doing wrong. Thanks in advance.

**Comment by neSpecc** [comment, marked as answer]
Use `div` instead of `textarea`

**Reply by msrumon** [reply]
Thanks. :)
